src/js/plugin.js

```diff
@@ -312,6 +312,9 @@ class Plugin {
     if (name !== BASE_PLUGIN_NAME) {
       if (Plugin.isBasic(plugin)) {
         Player.prototype[name] = createBasicPlugin(name, plugin);
+        Object.keys(plugin).forEach(function(prop) {
+          Player.prototype[name][prop] = plugin[prop];
+        });
```

Comment:
yeah calling the function that `createBasicPlugin` returns on player is actually what initializes. The question is do we want to copy the methods that are on the plugin right away (this is how it used to work in 5.x)? or only after the plugin has been initialized. 

Examples:

After initialization copy properties
```js
var basic = () => {};
basic.VERSION = '9.9.9';
videojs.registerPlugin('basic', basic);
...
console.log(player.basic.VERSION);
// undefined
player.basic();

console.log(player.basic.VERSION);
// 9.9.9
```

Before initialization copy properties
```js
var basic = () => {};
basic.VERSION = '9.9.9';
videojs.registerPlugin('basic', basic);
...
console.log(player.basic.VERSION);
// '9.9.9'
```